### PR TITLE
Clarify how to add block to page

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

The purpose of this pull request is to add a link to instructions that explain how to add a completed block to a page.  Also removed the conditional Page Builder instructions in this topic, added a callout at the top of the page that directs Page Builder users to the Page Builder block tutorial, and updated all screenshots.

## Affected documentation pages
https://docs.magento.com/m2/ce/user_guide/cms/block-add.html
https://docs.magento.com/m2/ee/user_guide/cms/block-add.html
https://docs.magento.com/m2/b2b/user_guide/cms/block-add.html

## Affected Magento editions

- [ x ] Open Source
- [ x ] Commerce
- [ x ] B2B

### CODE REVIEW

**Comment by dshevtsov** [comment]
@ccondra have you optimized the images after updating the screenshots?

File: src/cms/block-add.md [hunk 1]
```diff
@@ -31,47 +35,21 @@ _Blocks Workspace_
...
-<!--{% if "Default.EE-B2B" contains site.edition %}-->
-1. If you are using [Page Builder]({% link cms/page-builder.md %}), drag the content types that you need to the stage.
-
-    Then, follow the instructions for each content type.
-
-    - [Layout]({% link cms/page-builder-layout.md %}) (Row, Column, Tabs)
+1.  Use the [editor]({% link cms/editor.md %}) to format text, create links and tables, images, video, and audio.
 
-    - [Elements]({% link cms/page-builder-elements.md %}) (Text, Heading, Buttons, Divider, HTML Code)
+1.  If you prefer to work with HTML code, click <span class="btn">Show / Hide Editor</span>
```
Comment (leslietilling) [inline src/cms/block-add.md]:
```suggestion
1.  If you prefer to work with HTML code, click <span class="btn">Show / Hide Editor</span>.
```

**Comment by dshevtsov** [comment]
@leslietilling looks like @ccondra does not have enough permissions to self-assign to PRs
https://github.com/magento/merchdocs/pull/226/files#issuecomment-571740759

**Comment by ccondra** [comment]
Yes, the screenshots are optimized. Thanks!

**Comment by ccondra** [comment]
Running tests

**Pull request merged**